Having multiple .ftl files is causing an error

```
error[E0308]: mismatched types
  --> src/app.rs:35:5
   |
35 | /     leptos_fluent! {{
36 | |         locales: "./locales",
37 | |         translations: TRANSLATIONS,
38 | |         check_translations: "./src/**/*.rs",
...  |
45 | |         set_language_to_cookie: true,
46 | |     }};
   | |______^ expected an array with a fixed size of 52 elements, found one with 321 elements
```

This happens if I have both 'main.ftl' and `test.ftl` in locales/*/

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 2024-06-15 - [0.0.36]
+
+### Bug fixes
+
+- Fix error building files tracker when multiple files for each language.
+
 ## 2024-06-15 - [0.0.35]
 
 ### Bug fixes
@@ -192,6 +198,7 @@
 
 - Added all ISO-639-1 and ISO-639-2 languages.
 
+[0.0.36]: https://github.com/mondeja/leptos-fluent/compare/v0.0.35...v0.0.36
 [0.0.35]: https://github.com/mondeja/leptos-fluent/compare/v0.0.34...v0.0.35
 [0.0.34]: https://github.com/mondeja/leptos-fluent/compare/v0.0.33...v0.0.34
 [0.0.33]: https://github.com/mondeja/leptos-fluent/compare/v0.0.32...v0.0.33

--- a/README.md
+++ b/README.md
@@ -19,7 +19,7 @@ Add the following to your `Cargo.toml` file:
 
 ```toml
 [dependencies]
-leptos-fluent = "0.0.35"
+leptos-fluent = "0.0.36"
 fluent-templates = "0.9"
 
 [features]

--- a/examples/ssr-hydrate-actix/locales/en/404.ftl
+++ b/examples/ssr-hydrate-actix/locales/en/404.ftl
@@ -1,0 +1,1 @@
+not-found = Not Found

--- a/examples/ssr-hydrate-actix/locales/en/main.ftl
+++ b/examples/ssr-hydrate-actix/locales/en/main.ftl
@@ -1,2 +1,1 @@
 welcome-to-leptos = Welcome to Leptos!
-not-found = Not Found

--- a/examples/ssr-hydrate-actix/locales/es/404.ftl
+++ b/examples/ssr-hydrate-actix/locales/es/404.ftl
@@ -1,0 +1,1 @@
+not-found = No encontrado

--- a/examples/ssr-hydrate-actix/locales/es/main.ftl
+++ b/examples/ssr-hydrate-actix/locales/es/main.ftl
@@ -1,2 +1,1 @@
 welcome-to-leptos = ¡Bienvenido a Leptos!
-not-found = No encontrado

--- a/leptos-fluent-macros/Cargo.toml
+++ b/leptos-fluent-macros/Cargo.toml
@@ -2,7 +2,7 @@
 name = "leptos-fluent-macros"
 description = "Macros for leptos-fluent"
 edition.workspace = true
-version = "0.0.35"
+version = "0.0.36"
 license = "MIT"
 documentation.workspace = true
 repository.workspace = true

--- a/leptos-fluent-macros/src/files_tracker.rs
+++ b/leptos-fluent-macros/src/files_tracker.rs
@@ -7,15 +7,14 @@ pub(crate) fn build_files_tracker_quote(
 ) -> proc_macro2::TokenStream {
     let mut files_tracker_str = "{".to_string();
     for (lang, paths) in fluent_resources.iter() {
-        files_tracker_str
-            .push_str(&format!("let {} = vec![", lang.replace('-', "_")));
-        for path in paths {
+        for (i, path) in paths.iter().enumerate() {
             files_tracker_str.push_str(&format!(
-                "include_bytes!(\"{}\"),",
+                "let {}{} = include_bytes!(\"{}\");",
+                lang.replace('-', "_"),
+                i,
                 &escape_string(path)
             ));
         }
-        files_tracker_str.push_str("];");
     }
     if let Some(languages_file_path) = &languages_path {
         files_tracker_str.push_str(&format!(

--- a/leptos-fluent/Cargo.toml
+++ b/leptos-fluent/Cargo.toml
@@ -2,7 +2,7 @@
 name = "leptos-fluent"
 description = "Fluent framework for internationalization of Leptos applications"
 edition.workspace = true
-version = "0.0.35"
+version = "0.0.36"
 license = "MIT"
 documentation.workspace = true
 repository.workspace = true

--- a/leptos-fluent/README.md
+++ b/leptos-fluent/README.md
@@ -19,7 +19,7 @@ Add the following to your `Cargo.toml` file:
 
 ```toml
 [dependencies]
-leptos-fluent = "0.0.35"
+leptos-fluent = "0.0.36"
 fluent-templates = "0.9"
 
 [features]

--- a/leptos-fluent/src/lib.rs
+++ b/leptos-fluent/src/lib.rs
@@ -14,7 +14,7 @@
 //!
 //! ```toml
 //! [dependencies]
-//! leptos-fluent = "0.0.35"
+//! leptos-fluent = "0.0.36"
 //! fluent-templates = "0.9"
 //!
 //! [features]

